openstack/swift/charts/swift_http_importer/templates/configmap.yaml

```diff
@@ -0,0 +1,29 @@
+{{- range $repo, $config := .Values.repos }}
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: swift-http-import-{{$repo}}
+  labels:
+    system: openstack
+    component: repo
+
+data:
+  config.yaml: |
+    swift:
+      auth_url: {{$.Values.auth_url}}
+      user_name: {{$.Values.user_name}}
+      user_domain_name: {{$.Values.user_domain_name}}
+      project_name: {{$.Values.project_name}}
+      project_domain_name: {{$.Values.project_domain_name}}
+      password: {{$.Values.password}}
+      region_name: {{$.Values.global.region}}
+
+    jobs:
+      {{- range $index, $job := $config.jobs}}
+      - from: {{$job.source}}
+        to:   {{$job.target}}
+      {{- end}}
+
+---
+{{end}}
```

Comment:
You can put everything in one configmap by disambiguating on the name, i.e. `{{$repo}}.yaml` instead of `config.yaml`.